src/common/cuepoint/cuepoint-event-type.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+/**
+ * @const {Object} CuePointEventType
+ *
+ * @example
+ * // How to use
+ * player.addEventListener(KalturaPlayer.cuepoint.Event.TIMED_METADATA_ADDED, e => {
+ *   console.log(e.payload);
+ * };
+ */
+const CuePointEventType: {[event: string]: string} = {
+  /**
+   * Fires when new cue-points added to text-track.
+   * @event TIMED_METADATA_ADDED
+   * @memberof CuePointEventType
+   */
+  TIMED_METADATA_ADDED: 'timedmetadataadded',
+  /**
+   * Fired when the timed metadata triggered
+   * @event TIMED_METADATA
+   * @memberof CuePointEventType
+   */
+  TIMED_METADATA: 'timedmetadata'
```

Comment:
up until now `timedmetadata` was not for cuepoints and was part of core events. we can also add `timedmetadataadded` to it, no need to create additional nested/namespaced event object

Reply:
I created nested/namespaced event object because of PR comments that I received to the PR

Reply:
ok, so @yairans or @RoyBregman - `timedmetadata` is part of core already, so we don't need to move it and actually can't/shouldn't.
I suggest not to create a namespace for this only for 1 event and move it to the core library, not first time we ad an event in core for benefit of external use, when the bigger context makes more sense then defining it outside

Reply:
done